docs/sources/get-started/labels/_index.md

```diff
@@ -9,83 +9,283 @@ aliases:
 ---
 # Understand labels
 
-Labels are key value pairs and can be defined as anything! We like to refer to them as metadata to describe a log stream. If you are familiar with Prometheus, there are a few labels you are used to seeing like `job` and `instance`, and I will use those in the coming examples.
+Labels are a crucial part of Loki. They allow Loki to organize and group together log messages into log streams. Each log stream must have at least one label to be stored and queried in Loki.
 
-The scrape configs we provide with Grafana Loki define these labels, too. If you are using Prometheus, having consistent labels between Loki and Prometheus is one of Loki's superpowers, making it incredibly [easy to correlate your application metrics with your log data](/blog/2019/05/06/how-loki-correlates-metrics-and-logs--and-saves-you-money/).
+In this topic we'll learn about labels and why your choice of labels is important when shipping logs to Loki.
 
-## How Loki uses labels
+{{< admonition type="note" >}}
+Labels are intended to store [low-cardinality](https://grafana.com/docs/loki/<LOKI_VERSION>/get-started/labels/cardinality/) values that describe the source of your logs. If you frequently search high-cardinality data in your logs, you should use [structured metadata](https://grafana.com/docs/loki/<LOKI_VERSION>/get-started/labels/structured-metadata/).
+{{< /admonition >}}
+
+## Understand labels
+
+In Loki, the content of each log line is not indexed. Instead, log entries are grouped into streams which are indexed with labels.
+
+A label is a key-value pair, for example all of the following are labels:
+
+- deployment_environment = development
+- host = server-1
+- pod_name = myapp-abcdef
```

Comment:
`pod_name` is a tough one because I have seen users with unique hashes added to their podnames. In most cases if the name is static its fine. But there are cases like this:
```
NAME                                           READY   STATUS    RESTARTS   AGE
nginx-554b9c67f9-c5cv4                         1/1     Running   0          74s
nginx-554b9c67f9-hjkjq                         1/1     Running   0          74s
nginx-554b9c67f9-wbwdm                         1/1     Running   0          2m7s
```

Reply:
I see pod name frequently called out as a poor label choice in Slack, which is why I put it on the list.

Reply:
aaah but the way I read this section was these are suggested to be labels. I wouldn't have put pod_name in the list unless we are calling it the odd one out

Reply:
You're right, I missed that!  Will remove.